service/src/main/java/bio/terra/user/service/status/BaseStatusService.java

```diff
@@ -0,0 +1,91 @@
+package bio.terra.user.service.status;
+
+import bio.terra.user.app.configuration.StatusCheckConfiguration;
+import java.time.Instant;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.function.Supplier;
+import javax.annotation.PostConstruct;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BaseStatusService {
```

Comment:
again, are you expecting this to be used by more than statusService? If not, I don't think we need this super class.

Also based on my other comment, let just not have this service yet?